Dockerfile.modules

```diff
@@ -0,0 +1,14 @@
+FROM debian:jessie
```

Comment:
To answer your question, I don't see an official one, there are some random ones, though.